jax/_src/nn/functions.py

```diff
@@ -1210,81 +1210,206 @@ def _check_shape_and_dtype(t: Array | None, shape: Sequence[int],
   return jnp.reshape(out, output_shape)
 
 def scaled_matmul(
-    lhs: Array,
-    rhs: Array,
-    lhs_scales: Array,
-    rhs_scales: Array,
+    a: Array,
+    b: Array,
+    a_scales: Array,
+    b_scales: Array,
```

Comment:
Post-merge comment: changing the keyword argument names is a breaking change in a public API, and needs a deprecation cycle. We should revert this before it makes it into a release.